back/engines/commercial/analysis/app/models/analysis/auto_tagging_task.rb

```diff
@@ -1,5 +1,28 @@
 # frozen_string_literal: true
 
+# == Schema Information
+#
+# Table name: analysis_background_tasks
+#
+#  id                  :uuid             not null, primary key
+#  analysis_id         :uuid             not null
+#  type                :string           not null
+#  state               :string           not null
+#  progress            :float
+#  started_at          :datetime
+#  ended_at            :datetime
+#  auto_tagging_method :string
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+# Indexes
+#
+#  index_analysis_background_tasks_on_analysis_id  (analysis_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (analysis_id => analysis_analyses.id)
+#
```

Comment:
Unrelated auto-comments for models created by other dev (my run of `db:migrate` created the comments)